demos/whisper-base/main.js

```diff
@@ -105,6 +105,9 @@ let subAudioChunkLength = 0; // length of a sub audio chunk
 let subText = "";
 let speechToText = "";
 
+let time_to_first_token = 0; // TTFT
+let num_tokens = 0; // number of tokens
```

Comment:
Do we need to reset the num_tokens to 0 for a new inference session? Or I may miss something.

Reply:
Thanks, good catch, `num_tokens` should be reset.

Fixed and PTAL again.